modules/wildfly/pom.xml

```diff
@@ -61,11 +61,6 @@
             <optional>true</optional>
         </dependency>
         <dependency>
-            <groupId>org.hibernate</groupId>
```

Comment:
We don't need that anymore? Isn't it needed as part of putting the latest ORM version in place?